app/src/test/kotlin/com/intive/picover/parties/viewmodel/PartiesViewModelTest.kt

```diff
@@ -44,10 +48,47 @@ class PartiesViewModelTest : ShouldSpec(
 			).forAll { (result, state) ->
 				every { repository.parties() } returns result
 
-				tested = PartiesViewModel(repository)
+				tested = PartiesViewModel(
+					partiesRepository = repository,
+					shortTextValidator = textValidator,
+					longTextValidator = textValidator,
+				)
 
 				tested.state.value shouldBe state
 			}
 		}
+
+		should("call validator WHEN validateShortInputText runs") {
+			val text = "MyUserName1234"
+			every { textValidator.validate(text) } returns ValidationStatus.ValidText
+
+			tested.validateShortText(text)
+
+			verify { textValidator.validate(text) }
+		}
+
+		should("call validator WHEN validateLongText runs") {
+			val text = "MyUserName1234"
+			every { textValidator.validate(text) } returns ValidationStatus.ValidText
+
+			tested.validateLongText(text)
+
+			verify { textValidator.validate(text) }
+		}
+
+		should("update the title WHEN newTitle is provided") {
+			val title = "new title"
+
+			tested.updateTitle(title)
+
+			tested.title shouldBe title
+		}
+
+		should("update the description WHEN newDescription is provided") {
+			val description = "new description"
```

Comment:
Minor: Empty line between given when and then is missing :)

Reply:
Done